topbeat/docs/fields.asciidoc

```diff
@@ -352,6 +352,13 @@ type: string
 The process name.
 
 
+==== proc.username
+
+type: string
+
+The user name of the process owner. Added in 1.2.1.
```

Comment:
@dedemorton I think Clint mentioned at some point that there is some special asciidoc syntax for marking things that are added in a new version, but I'm not quite sure. Do you know?